Jenkinsfile

```diff
@@ -21,7 +17,18 @@ pipeline {
         }
         stage('Unit Tests') {
             steps {
-                sh 'build/run make -j\$(nproc) check'
+                sh 'build/run make -j\$(nproc) test'
+            }
+        }
+        stage('Integration Tests') {
+            environment {
+                KUBE_VERSION = "v1.6"
+                SUITE = "BlockStorage"
+            }
+            steps {
+                sh 'env | sort'
+                sh 'tests/scripts/kube-dind-cluster.sh up'
```

Comment:
are the integration tests and the kube-dind-cluster running inside the build container?
If not we should probably do a 'kube-dind-cluster.sh clean'  after test runs.